### PR TITLE
[somfytahoma] New method to send a command to several devices in the same place

### PR DESCRIPTION
**Pull request by lolodomo**
Also include the place into the inbox label (discovery)

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/discovery/SomfyTahomaItemDiscoveryService.java [hunk 1]
```diff
@@ -366,8 +386,17 @@ private boolean hasCommmand(SomfyTahomaDevice device, String command) {
...
     }
 
-    private void deviceDiscovered(SomfyTahomaDevice device, ThingTypeUID thingTypeUID) {
-        deviceDiscovered(device.getLabel(), device.getDeviceURL(), device.getOid(), thingTypeUID,
+    private void deviceDiscovered(SomfyTahomaDevice device, ThingTypeUID thingTypeUID, @Nullable String place) {
+        deviceDiscovered(null, device, thingTypeUID, place);
+    }
+
+    private void deviceDiscovered(@Nullable String otherLabel, SomfyTahomaDevice device, ThingTypeUID thingTypeUID,
+            @Nullable String place) {
```
Comment (octa22) [inline bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/discovery/SomfyTahomaItemDiscoveryService.java:394]: Why is here the method ```deviceDiscovered``` with first parameter ```@Nullable String otherLabel```, I see calling only with the ```null``` value. In this case merge this method with ```private void deviceDiscovered(SomfyTahomaDevice device, ThingTypeUID thingTypeUID, @Nullable String place)``` and have one method instead.

**Review by octa22 — COMMENTED** [review]
Generally I am asking myself why we should have this code increment, if it does not bring any new channel, so there is no added value of this code for users. Sorry if I am missing something, but I would at lease expect a way how to invoke this method ```sendCommandToSameDevicesInPlace```

File: bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/discovery/SomfyTahomaItemDiscoveryService.java (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/discovery/SomfyTahomaItemDiscoveryService.java:394]: I was expecting this remark ;)
I used it in coming code (next PR).
But I could introduce it only in my next PR.

**Comment by lolodomo** [comment]
I try to provide small PR to move to my final PR for Cozytouch integration.
This PR adds at least one change for the users, the room is now part of the inbox thing label.

File: bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/discovery/SomfyTahomaItemDiscoveryService.java (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/discovery/SomfyTahomaItemDiscoveryService.java]: Fixed. Method removed.

**Comment by lolodomo** [comment]
Note that you will be able later to enable the "room" feature for certain Tahoma devices the same way I do it for Cozytouch radiators. But it has to be thought (by you) on what devices it makes really sense.

**Comment by octa22** [comment]
OK, I understand

**Review by octa22 — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
@fwolter : please review

**Comment by lolodomo** [comment]
@openhab/add-ons-maintainers : please review & merge.

**Review by fwolter — APPROVED** [review]
LGTM

**Pull request merged**